ArgentumRhodon/BatBotsWebsite
# Home Page Layout

Changes:
- Added a placeholder layout for the homepage content
- Improved navbar (added mobile dropdown animation, title link, and moved "home" into mobile dropdown)
- Added footer (WIP)

Test by opening the latest Pages deployment and:
- View in mobile and desktop mode to test responsiveness
- View in multiple browsers if possible to check consistency

Issue: https://bat-bots.atlassian.net/jira/software/projects/BB/boards/1?selectedIssue=BB-118

Due in Sprint 2
